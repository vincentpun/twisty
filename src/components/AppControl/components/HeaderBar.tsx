import * as React from 'react';

interface HeaderBarProps {
  title: string;
}

const HeaderBar = ({ title }: HeaderBarProps) => (
  <nav className="app-control__header-bar">
    <span className="app-control__header-bar-title">{title}</span>
  </nav>
);

export default HeaderBar;
