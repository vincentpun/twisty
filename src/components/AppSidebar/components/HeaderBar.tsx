import * as React from 'react';

interface HeaderBarProps {
  title: string;
}

const HeaderBar = ({ title }: HeaderBarProps) => (
  <nav className="navbar navbar-dark bg-primary">
    <span className="navbar-brand">{title}</span>
  </nav>
);

export default HeaderBar;
