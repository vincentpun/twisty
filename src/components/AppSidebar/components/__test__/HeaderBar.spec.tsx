import * as React from 'react';
import { shallow } from 'enzyme';

import HeaderBar from '../HeaderBar';

describe('HeaderBar', () => {
  it('matches the snapshot', () => {
    const wrapper = shallow(<HeaderBar title="" />);
    expect(wrapper).toMatchSnapshot();
  });

  it('renders the title', () => {
    const wrapper = shallow(<HeaderBar title="Hero" />);
    expect(wrapper.find('.navbar-brand').text()).toEqual('Hero');
  });
});
