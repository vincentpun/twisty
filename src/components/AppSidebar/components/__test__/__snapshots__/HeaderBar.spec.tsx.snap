// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderBar matches the snapshot 1`] = `
<nav
  className="navbar navbar-dark bg-primary"
>
  <span
    className="navbar-brand"
  />
</nav>
`;
